polaris.shopify.com/package.json

```diff
@@ -85,13 +92,23 @@
     "js-yaml": "^4.1.0",
     "lodash.set": "^4.3.2",
     "marked": "^4.0.16",
+    "node-fetch": "^3.3.1",
     "ora": "^6.2.0",
     "p-map": "^5.5.0",
-    "playroom": "^0.28.0",
+    "playroom": "0.28.0",
+    "playwright-core": "^1.31.2",
     "puppeteer": "^16.0.0",
     "sass": "^1.49.9",
     "style-loader": "^3.3.1",
+    "tiny-invariant": "^1.3.1",
+    "tmp": "^0.2.1",
     "ts-node": "^10.7.0",
-    "typescript": "^4.9.3"
+    "typescript": "^4.9.3",
+    "yargs": "^17.7.1"
+  },
+  "resolutions": {
+    "@types/react": "18.2.0",
+    "@types/react-dom": "18.2.0",
+    "playroom": "0.28.0"
```

Comment:
Have to lock down the version since a new one got released which is conflicting with our patch to it.